### ISSUE TITLE
Issue 5: Normalización de Datos (F1)

### ISSUE THREAD

**Issue by ardillaCHIKI** [ardillaCHIKI/El-Or-culo-del-Tiempo#5]
Descripción:
Escalar los valores del consumo diario al rango [0, 1] usando MinMaxScaler (de sklearn.preprocessing).

Tareas:

 Aplicar el escalado a la columna de consumo.

 Guardar los parámetros de normalización (min, max) para uso posterior.

 Verificar que los datos transformados estén en el rango correcto.

Criterio de aceptación:
Los valores del dataset normalizado se encuentran dentro del rango [0, 1].
